exspy/tests/components/test_double_power_law.py

```diff
@@ -46,11 +46,11 @@ def test_function():
 
 class TestDoublePowerLaw:
     def setup_method(self, method):
+        pytest.importorskip("exspy")
```

Comment:
In what situation would `exspy` not be available for its own tests?

Reply:
Indeed, it doesn't make sense!